lessons/operator/02/README.md

```diff
@@ -1,257 +1,267 @@
-Labels and annotations are identical in format &ndash; they are represented as `"key": "value"` pairs, and all values must be strings &ndash; but they serve different purposes.
-
-- **Labels**
-
-  Labels are indexed by Sensu and used as selectors (e.g. for "filtering" resources).
-  Good examples of labels include "environment" or "region" – metadata that can be used to group or select subsets of resources.
-
-- **Annotations**
-
-  Annotations are for storing additional data that may be used downstream in the Sensu observability pipeline, or by an external system that Sensu forwards data to.
-  Good examples of annotations include things like configuration data (e.g. what Slack channel should Sensu alert in for a given entity/check/event), or other structured data (e.g. stringified JSON) containing instructions for third-party tools.
-  Annotations are not indexed by Sensu, so they should not be used for grouping/selecting purposes.
-
-<details>
-<summary><strong>Example Sensu event <code>metadata</code> object:</strong></summary>
-
-```json
-{
-  "metadata": {
-    "labels": {
-      "region": "us-west-2"
-    },
-    "annotations": {
-      "sensu.io/plugins/slack/config/channel": "#dev"
-    }
-  },
-  "entity": {},
-  "check": {},
-  "metrics": {},
-  "id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-  "timestamp": 1234567890
-}
-```
-
-_NOTE: in Sensu Go, almost every API resource supports `metadata`, typically including `metadata.labels`, `metadata.annotations`, and a `metadata.name` field.
-As a result, Sensu Events may contain top-level event metadata as shown above, as well as Check metadata and Entity metadata._
-
-</details>
-
-## Learn more
-
-- [[Documentation] "Sensu Events Reference" (docs.sensu.io)](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/)
-- [[Documentation] "What is Observability?" (sumologic.com)](https://www.sumologic.com/glossary/observability/)
-- [[Blog Post] "Monitoring and Observability", by Cindy Sridharan (copyconstruct.medium.com)](https://copyconstruct.medium.com/monitoring-and-observability-8417d1952e1c)
-- [[Blog Post] "Monitoring versus Observability" (thenewstack.io)](https://thenewstack.io/monitoring-vs-observability-whats-the-difference/)
-
-## Next steps
+# Lesson 2: Setup the Workshop Sandbox
+
+- [Goals](#goals)
+- [Prerequisites](#prerequisites)
+- [The Workshop Environment](#the-workshop-environment)
+- [How It All Works Together](#how-it-all-works-together)
+- [EXERCISE 1: Install the Workshop](#exercise-1-install-the-workshop)
+- [Discussion](#discussion)
+- [Learn More](#learn-more)
+- [Next Steps](#next-steps)
+
+## Goals
+
+In this lesson we will setup a local multi-node Sensu environment for use during the workshop.
+This lesson is intended for operators of Sensu or anyone who would like to explore Sensu from a technical perspective.
+
+## Prerequisites
```

Comment:
There's one other prerequisite that might be worth calling out that I haven't thought of before now: port conflicts. 

This workshop will attempt to bind services on ports 3000, 8080, 8081, 139, 445, and 3001 (if Grafana is deployed).

Reply:
Good point. This may be too much detail for this list though. I think that makes more sense to include in troubleshooting guide (e.g. I tried to launch the workshop environment, but it failed to launch.. why? oh, a port conflict, ok, lets free up that port or change the docker configs slightly).

Reply:
Let's add an issue to resolve this later.